pkg/k8s/service_cache.go

```diff
@@ -114,11 +115,17 @@ func (s *ServiceCache) GetServiceIP(svcID ServiceID) *loadbalancer.L3n4Addr {
 	s.mutex.RLock()
 	defer s.mutex.RUnlock()
 	svc := s.services[svcID]
-	if svc == nil {
+	if svc == nil || len(svc.FrontendIPs) == 0 || len(svc.Ports) == 0 {
 		return nil
 	}
+
+	feIP := ip.GetIPFromListByFamily(svc.FrontendIPs, option.Config.EnableIPv4)
```

Comment:
As it is nicely documented elsewhere:
> `GetIPFromListByFamily` returns a single IP address of the provided family from a list

Is it possible to set multiple IPs of the same family in the `.FrontendIPs` field?
If yes, should we have a selection here that will not always select the same?

Reply:
> Is it possible to set multiple IPs of the same family in the .FrontendIPs field?
I don't think so. Only a single ClusterIP (VIP) per family should be allowed.